scripts/globals/spells/damage_spell.lua

```diff
@@ -580,16 +598,7 @@ xi.spells.damage.calculateResist = function(caster, target, spell, skillType, sp
     end
 
     resistTier = utils.clamp(resistTier, 0, 3)
-
-    if resistTier == 0 then     -- Unresisted
-        resist = 1
-    elseif resistTier == 1 then -- (1/2)
-        resist = 0.5
-    elseif resistTier == 2 then -- (1/4)
-        resist = 0.25
-    elseif resistTier == 3 then -- (1/8)
-        resist = 0.125
-    end
+    resist     = 1 / (2 ^ resistTier)
```

Comment:
Nice

Reply:
I recall this originally coming in that way, before some past dev decided it was clearer to just if/elseif it